### PR TITLE
Changes createsurvey class to have one list for all questions 

### PR DESCRIPTION
**Pull request by hassanidea**
Used the abstract class QType for all question types in createSurvey

### CODE REVIEW

File: src/main/java/com/SurveyMonkey/controller/DropDownQuestionController.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DropDownQuestionController {
+    List<DropDownQuestion> dropDownQuestions = new ArrayList<>();
```
Comment (TaherShabaan99) [inline src/main/java/com/SurveyMonkey/controller/DropDownQuestionController.java:15]: This list should be of type "QType" so it should be
List<QType> qtype = new ArrayList<>();

File: src/main/java/com/SurveyMonkey/controller/DropDownQuestionController.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    @Autowired
+    private CreateSurveyRepository createSurveyRepository;
+    @Autowired
+    private DropDownRepository dropDownRepository;
+
+
+    @PostMapping("/saveDropDownQuestion")
+    public ResponseEntity<Object> addQuestion(@RequestBody DropDownQuestion dropDown) {
+        dropDownQuestions.add(dropDown);
```
Comment (TaherShabaan99) [inline src/main/java/com/SurveyMonkey/controller/DropDownQuestionController.java:24]: This should be:
 qtype.add(dropDown);

File: src/main/java/com/SurveyMonkey/controller/DropDownQuestionController.java (hunk 1 above)
Comment (TaherShabaan99) [inline src/main/java/com/SurveyMonkey/controller/DropDownQuestionController.java:15]: This list should be of type "QType" so it should be
List<QType> qtype = new ArrayList<>();

Comment (TaherShabaan99) [inline src/main/java/com/SurveyMonkey/controller/DropDownQuestionController.java:15]: change the type of the List to be of type QType

File: src/main/java/com/SurveyMonkey/controller/DropDownQuestionController.java (hunk 2 above)
Comment (hassanidea) [reply src/main/java/com/SurveyMonkey/controller/DropDownQuestionController.java:25]: Made the required changes it is now QType

**Comment by TaherShabaan99** [comment]
Great that you changed the types and it looks good to me!

**Review by TaherShabaan99 — APPROVED** [review]
Looks good to me!

**Comment by Dez1999** [comment]
Accepted! Looks good now

**Review by Dez1999 — APPROVED** [review]
Conflicts have been resolved, so this is approved

**Review by OmarAgamy9 — APPROVED** [review]
Good changes! Go ahead

**Pull request merged**